Jenkinsfile

```diff
@@ -13,8 +13,8 @@ pipeline {
     lib('fxtest@1.10')
   }
   triggers {
-    pollSCM('H/5 * * * *')
-    cron('H H * * *')
+    pollSCM(env.BRANCH_NAME == 'master' ? 'H/5 * * * *' : '')
```

Comment:
For what it's worth, multibranch pipeline will scan the repo and if it notices a chance it will trigger a build. Do we need to poll every five minutes for changes, or could the scan frequency be enough? We could even reduce the scan frequency.

Reply:
On second thoughts, mutlibranch is only in use on preprod, so maybe let's keep it.